app/src/off/res/values/strings.xml

```diff
@@ -10,6 +10,7 @@
     <string name="website_product">https://world.openfoodfacts.org/product/</string>
     <!-- For fr, Replace by https://world-fr.openfoodfacts.org/contributeur/ -->
     <string name="website_contributor">https://world.openfoodfacts.org/contributor/</string>
+    <string name="txtHomeOnline">Open Food Facts is an open database of more than %1$d food products (plus the ones you will contribute) with ingredients, allergens, nutrition facts and all the tidbits of information we can find on product labels.</string>
```

Comment:
Add the string resource to all the flavors of the app and not just OFF. The build is failing for other flavors because of this.
```
androidapp/app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/HomeFragment.java:73: error: cannot find symbol
                        String txtHomeOnline = getResources().getString(R.string.txtHomeOnline);
```

Reply:
Just add three more strings similar to the one added for OFF. The base URL is different for all the flavors and is picked up automatically depending upon the build flavor you shouldn't be worrying about that. 

Add strings to the below paths 

`app/src/obf/res/values/strings.xml'
`app/src/opf/res/values/strings.xml'
`app/src/opff/res/values/strings.xml'

Reply:
@huzaifaiftikhar Did those changes in the latest push.

![screenshot_1537774824](https://user-images.githubusercontent.com/32436867/45941656-dfab2800-bffc-11e8-8c6d-cb18ce87a56d.png)
![screenshot_1537775036](https://user-images.githubusercontent.com/32436867/45941657-dfab2800-bffc-11e8-82e2-5bc186947825.png)
![screenshot_1537775311](https://user-images.githubusercontent.com/32436867/45941658-e043be80-bffc-11e8-99a7-aef36b26281f.png)

Reply:
@huzaifaiftikhar But there's one more thing if I change the language to francis or any other, changes I made will always be shown in en language.

![screenshot_1537775702](https://user-images.githubusercontent.com/32436867/45941812-7b3c9880-bffd-11e8-8037-08667b4de339.png)

Reply:
No issues, we'll get the translations via crowdin.

Reply:
@huzaifaiftikhar So I guess as soon as the Travis build finished successfully, this PR is good to go. Right?